src/pyxdf/pyxdf.py

```diff
@@ -202,6 +220,9 @@ def load_xdf(
 
     logger.info("Importing XDF file %s..." % filename)
 
+    # Validate handle_non_monotonic argument.
+    handle_non_monotonic = HandleNonMonoState(handle_non_monotonic)
```

Comment:
Although this validates the input, the error message you get for an unsupported value is not very helpful (`ValueError: 'whatever' is not a valid HandleNonMonoState`). I would handle (no pun intended) this with a custom more specific error message, which probably makes the use of an `Enum` unnecessary.